cmd/podman/ps.go

```diff
@@ -349,22 +345,15 @@ func psCmd(c *cli.Context) error {
 		}
 	} else {
 		// Output Namespace headers
-		fmt.Fprintf(w, "\n%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s", hid, hnames, nspid, nscgroup, nsipc, nsmnt, nsnet, nspidns, nsuserns, nsuts)
-	}
-	if len(pss) == 0 {
```

Comment:
this should probably have stayed in

Reply:
it was moved down for both conditions now

Reply:
I meant the if for the heading

Reply:
Oh, that makes perfect sense - SGTM